templates/vue/src/components/modals/NodeModal/forms/ContentForm/AnswerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div>
+    <b-form-group data-qa="activity-combobox" label="Select an activity:">
+      <combobox
+        v-model="activityNodeId"
+        :options="currentActivityNodes"
+        data-qa="choose-activity-node"
+        item-text="title"
+        item-value="id"
+        empty-message="There are no activities yet."
+      >
+        <template v-slot="slotProps">
+          <p>
+            {{ slotProps.option.title }}
+          </p>
+        </template>
+      </combobox>
+    </b-form-group>
+    <b-form-group
+      data-qa="question-select"
+      label="Select a question from that activity:"
+    >
+      <combobox
+        v-model="currentQuestion"
+        :options="getCurrentQuestions"
+        data-qa="choose-question"
+        item-text="text"
+        item-value="id"
+        empty-message="There is no activity selected yet."
+      >
+        <template v-slot="slotProps">
+          <p>
+            {{ slotProps.option.text }}
+          </p>
+        </template>
+      </combobox>
+    </b-form-group>
+    <b-form-group
+      label="Show this text first: "
+      description="If empty, will default to the follow-up text of the activity form."

```

Comment:
Instead of this, can we show the actual question text in the placeholder for this input?

Reply:
I think we should also flip their order. First select a question, then we will show they questions text as a placeholder. What do you think?
![Screenshot from 2021-06-24 17-17-31](https://user-images.githubusercontent.com/50620330/123350576-21e85780-d510-11eb-92ce-a8897bb3a809.png)